MAINTAINERS.md

```diff
@@ -0,0 +1,20 @@
+The current maintainers of this repo are [@adamralph](https://github.com/adamralph), [@distantcam](https://github.com/distantcam), and [@HEskandari](https://github.com/HEskandari).
+
+The maintainers [watch](https://github.com/Particular/ServiceInsight/watchers) this repo and undertake the following responsibilities:
+
+- Ensuring that the `master` branch is always releasable. "Releasable" means that the software built from the latest commit can be released immediately simply by following the [release procedure](https://github.com/Particular/ServiceInsight/blob/master/docs/releasing.md).
```

Comment:
Yeah the idea of "releasable" needs to be fleshed out. By that definition the Pirac merge was "releasable".